docs/UserGuide.md

```diff
@@ -210,16 +210,23 @@ Format: `find KEYWORD [MORE_KEYWORDS]`
    * Persons matching at least one keyword will be returned (i.e. `OR` search).
      e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
 2. In `SCHEDULE` list:
-    * TBC
+    * The search is case-insensitive. e.g `lesson` will match `Lesson`
+    * The order of the keywords does not matter. e.g. `Lesson Chemistry` will match `Chemistry Lesson`
```

Comment:
It does matter, it checks the entire search string to see if lesson name contains the search string

Reply:
Thank you, made changes in new commit